include/wlr/render/gles2.h

```diff
@@ -30,8 +30,17 @@ struct wlr_gles2_texture_attribs {
 	bool has_alpha;
 };
 
+struct wlr_buffer;
+struct wlr_gles2_buffer_attribs {
+	GLuint fbo;
+};
+
+bool wlr_renderer_is_gles2(struct wlr_renderer *wlr_renderer);
 bool wlr_texture_is_gles2(struct wlr_texture *texture);
 void wlr_gles2_texture_get_attribs(struct wlr_texture *texture,
 	struct wlr_gles2_texture_attribs *attribs);
+void wlr_gles2_buffer_get_attribs(struct wlr_renderer *wlr_renderer,
+	struct wlr_buffer *wlr_buffer,
```

Comment:
Hm, this potentially operates on a buffer which isn't being used as a render target (via `wlr_renderer_bind_buffer`)... Is there a real use-case, or is it fine to operate on the currently bound buffer?

Reply:
What I can think of now is that it is only used after wlr_output_attach_render, because wlr_renderer_bind_buffer is the internal behavior of wlr_output.

Reply:
Done